tests/TestEntry.cpp

```diff
@@ -130,3 +131,28 @@ void TestEntry::testClone()
 
     delete entryOrg;
 }
+
+void TestEntry::testResolveUrl()
+{
+    Entry* entry = new Entry();
```

Comment:
This should be ```delete```d? Or maybe make resolveUrl a static function.

Reply:
It doesn't matter in a test. Technically it should be but it really doesn't matter.

Reply:
ok made the change 😝

Reply:
QUrl may be usable, but WITH_XC_HTTP is also for those people who don't like networking functionality inside their password manager's code.

Reply:
There is no networking code in here.

Reply:
True. I though, QUrl would also do some resolver work. But I guess I've just been doing to much Java as of late. :-)

Reply:
Was thinking the same thing when I first write that piece of code :D